### PR TITLE
More Doc Links

### PR DESCRIPTION
**Pull request by ben-directai**
Summary:
readme edits for clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,37 @@
-# DirectAI
+# [DirectAI](https://calendly.com/directai/demo)
```
Comment (ben-directai) [inline README.md]: this might look kind of ugly. any strong opinions? I can shift it to another location if you think it'd be better.

Comment (isaacrob) [reply README.md]: I agree it's kind of ugly .. already linking the Calendly at the end, I think it's fine to leave it out here

Comment (isaacrob) [reply README.md]: maybe replace it with a link to the website? I notice we're not linking that anywhere ?

**Review by isaacrob — APPROVED** [review]
lgtm

**Pull request merged**